pkgs/development/python-modules/jupysql/default.nix

```diff
@@ -84,6 +84,21 @@ buildPythonPackage rec {
   disabledTests = [
     # AttributeError: 'DataFrame' object has no attribute 'frame_equal'
     "test_resultset_polars_dataframe"
+    # all of these are broken with later versions of duckdb; see
+    "test_resultset_getitem"
+    "test_resultset_dict"
+    "test_resultset_len"
+    "test_resultset_dicts"
+    "test_resultset_dataframe"
+    "test_resultset_csv"
+    "test_resultset_str"
+    "test_resultset_repr_html_when_feedback_is_2"
+    "test_resultset_repr_html_with_reduced_feedback"
+    "test_invalid_operation_error"
+    "test_resultset_config_autolimit_dict"
+    # https://github.com/ploomber/jupysql/issues/1030
```

Comment:
Did you mean to put the link next to the other comment above the new entries?

Reply:
Yep, thanks 😄

Reply:
Fixed.